modules/twinkleprotect.js

```diff
@@ -640,6 +640,8 @@ Twinkle.protect.protectionTypes = [
 		list: [
 			{ label: 'Generic (semi)', value: 'pp-semi-protected' },
 			{ label: 'Persistent vandalism (semi)', selected: true, value: 'pp-semi-vandalism' },
+			{ label: 'Persistent disruptive editing (semi)', value: 'pp-semi-disruptive' },
+			{ label: 'Persistent addition of unsourced content (semi)', value: 'pp-semi-unsourced' },
```

Comment:
Keep it short. "Disruptive editing" and "Adding unsourced content" should be enough. The admin will see the actual protection log reason for themselves when they scroll down to have a look.